COPYRIGHT.txt

```diff
@@ -178,9 +178,14 @@ License: Apache-2.0
 
 Files: ./thirdparty/basis_universal/
 Comment: Basis Universal
-Copyright: 2022, Binomial LLC.
+Copyright: 2022-2024, Binomial LLC.
```

Comment:
Seems like the current files mostly have `2019-2024` copyright. Not sure where I got the sole `2022` originally.
```suggestion
Copyright: 2019-2024, Binomial LLC.
```